docs/standard/data/xml/xslt-transformations.md

```diff
@@ -18,9 +18,9 @@ ms.workload:
   - "dotnetcore"
 ---
 # XSLT Transformations
-The Extensible Stylesheet Language Transformation (XSLT) lets you transform the content of a source XML document into another document that is different in format or structure. For example, you can use XSLT to transform XML into HTML for use on a Web site or to transform it into a document that contains only the fields required by an application. This transformation process is specified by the [W3C XSL Transformations (XSLT) Version 1.0 recommendation](http://go.microsoft.com/fwlink/?LinkID=49919).  
+The Extensible Stylesheet Language Transformation (XSLT) lets you transform the content of a source XML document into another document that is different in format or structure. For example, you can use XSLT to transform XML into HTML for use on a Web site or to transform it into a document that contains only the fields required by an application. This transformation process is specified by the [W3C XSL Transformations (XSLT) recommendation](https://www.w3.org/TR/xslt/).  
   
- The <xref:System.Xml.Xsl.XslCompiledTransform> class is the XSLT processor in the .NET Framework. The <xref:System.Xml.Xsl.XslCompiledTransform> class supports the W3C XSLT 1.0 recommendation.  
+ The <xref:System.Xml.Xsl.XslCompiledTransform> class is the XSLT processor in .NET. The <xref:System.Xml.Xsl.XslCompiledTransform> class supports the [W3C XSLT 1.0 recommendation](https://www.w3.org/TR/xslt-10/).  
```

Comment:
@krwq, @pjanotti here and in other places of the docs we mention that XslCompiledTransform supports the XSLT 1.0 syntax. However, XSLT is now on version 3.0. Are we still only supporting version 1.0 or do we support other versions too?

Reply:
Only 1.0, see https://github.com/dotnet/corefx/issues/2295